cmd/scaffold.go

```diff
@@ -751,7 +751,7 @@ func (fnb *FlowNodeBuilder) ParseAndPrintFlags() error {
 		fnb.Logger.Fatal().Err(err).Msg("flow configuration validation failed")
 	}
 
-	info := fnb.Logger.Info()
+	info := fnb.Logger.Error()
```

Comment:
To avoid confusion, I would suggest to be very clear that this is _not_ an error but only logged at error level for visibility. Would suggest to extend the message in [line 757](https://github.com/onflow/flow-go/blob/2b2f6dd78e6ff8f924097f38eb72658f4822729b/cmd/scaffold.go#L757) to read `"configuration loaded (this is not an error; only logged at error level for visibility)"`